db/migrate/20200428064825_create_tag_taggables.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTagTaggables < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tag_taggables do |t|
+      t.belongs_to :taggable, polymorphic: true, index: true
+      t.belongs_to :tag, polymorphic: true, index: true
```

Comment:
Is this supposed to be polymorphic?

Reply:
Good spot. Yes it should not be polymorphic